### PR TITLE
Replace +++ in headers with ---

### LINKED ISSUE #70: Switch all file headers to use YAML
We have a mix of YAML (`---`) and TOML (`+++`) in page headers. Let's switch them all to use YAML for consistency's sake.

### PR DESCRIPTION
**Pull request by Hanzei**
Fixes #70

### CODE REVIEW

**Comment by Hanzei** [comment]
The build failed, but I'm quite sure its not related to my changes.
```
go run scripts/devtalks.go > site/data/DevTalks.json
2018/01/23 13:34:21 Environment variable YOUTUBE_API_KEY must be set in order to fetch dev talks
exit status 1
make: *** [devtalks-data] Error 1
```

**Comment by jwilander** [comment]
@MusikPolice any idea why the build is failing on that?

File: site/content/contribute/devtalks/_index.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-+++
+---
 title = "Mattermost Dev Talks"
```
Comment (jwilander) [inline site/content/contribute/devtalks/_index.md]: I think you also need to change the `=`s to `:`s, so that's it YAML formatted

Comment (Hanzei) [reply site/content/contribute/devtalks/_index.md]: Your right. I will fix this.

**Comment by Hanzei** [comment]
Since we use YAML in all files, should we use it also for the config file which is using `toml` atm?

**Comment by MusikPolice** [comment]
@jwilander I'm not sure. The environment variable is set in the build configuration.

**Comment by MusikPolice** [comment]
Hmm the build issue might be because the environment variable is secret, and @Hanzei is pushing the PR from a fork of the repository. [This documentation](https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings) seems to imply that secret environment variables won't be applied to builds of repository forks: 
> By default, the value of these new environment variables is hidden from the export line in the logs. This corresponds to the behavior of encrypted variables in your .travis.yml. The variables are stored encrypted in our systems, and get decrypted when the build script is generated.
> Similarly, we do not provide these values to untrusted builds, triggered by pull requests from another repository.
Suspect that even if that script passed, we would fail to deploy to AWS

**Comment by jwilander** [comment]
I'll test it locally quickly and then merge

**Comment by jwilander** [comment]
Works great 👍

**Pull request merged**